go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/facette/natsort v0.0.0-20181210072756-2cd4dd1e2dcb
 	github.com/google/uuid v1.4.0
 	github.com/goradd/maps v0.1.5
-	github.com/inspektor-gadget/inspektor-gadget v0.23.0
+	github.com/inspektor-gadget/inspektor-gadget v0.23.1
```

Comment:
why?

Reply:
this is the release they did for me, fixing dns tracing on minikube

Reply:
instead of using my fork